### DISCUSSION TITLE
Converting Days since 00-Jan-0000 00:00:00 to datetime

### DISCUSSION THREAD

**Discussion by vic1309** [SciTools/cf-units#212] [Q&A]
Dear all,

I have a dataset which contains the time variable defined as Days since 00-Jan-0000 00:00:00. A while ago, @ocefpaf [suggested](https://github.com/Unidata/netcdf4-python/issues/442#issuecomment-129059576) a workaround. However, it looks like it is not working as back in the days. I get the following error when I try to reproduce his example and with my data as well:

    TypeError: num2date() got an unexpected keyword argument 'has_year_zero'

Does anyone know how to solve that?

Cheers,
Victor

**Comment by trexfeathers** [comment]
Hi @vic1309, sorry for the delay.

`has_year_zero` is an argument introduced to `cftime.num2date()` in cftime `v1.5.0` (2021-05-20). cf-units uses cftime in many places, but does not use the `has_year_zero` argument anywhere, so I can't see where this error might have come from.

Do you have the latest versions of cf-units and cftime in your environment?
